rllib/tests/test_algorithm_save_load_checkpoint_learner.py

```diff
@@ -0,0 +1,90 @@
+import tempfile
+import unittest
+
+import ray
+from ray.rllib.utils.test_utils import check, framework_iterator
+from ray.rllib.algorithms.ppo import PPOConfig
+from ray.rllib.policy.sample_batch import DEFAULT_POLICY_ID
+from ray.rllib.utils.metrics.learner_info import LEARNER_INFO, LEARNER_STATS_KEY
+
+
+algorithms_and_configs = {
+    "PPO": (PPOConfig().training(train_batch_size=2, sgd_minibatch_size=2))
+}
+
+
+@ray.remote
```

Comment:
why do these tasks need to become ray tasks?

Reply:
it prevents the problem of setting seeding multiple times inside of the main process which would render all of the results seed reproducible.

Reply:
added a comment in the docstring